pages/learn/deploy/delta.md

```diff
@@ -6,29 +6,33 @@ thumbnail: /img/runtime/ResinSupervisorDelta.png
+When a new version of your application is pushed to {{ $names.company.lower }} servers, your devices are notified and will initiate an update of their running containers. The regular device update uses the Docker pull mechanism. It requests all the [layers](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/#/images-and-layers) of the new container image that are not present on the device (i.e. not shared with the previous image). Then from these layers, the new image is assembled on the device and replaced the previous version of the application. This process potentially moves a lot of data, uses a lot of space on the device to hold both the old and the new images, and the device can be in a sensitive state while Docker is updating (e.g. in case there's an unexpected power outage during that time).
+
+To address some of these issues, we have implemented a "binary delta" update process. Instead of initiating a Docker pull when the device is notified about an update, it requests the {{ $names.company.lower }} servers to provide just the differences between the old and new container image. This comparison is made on the full image level, comparing the actual content, regardless of the layers used, resulting in the minimum amount of change required to get from the previous application version to the new one. In the worst case (i.e. completely replaced application image), the binary delta is equal size to the Docker pull (the device needs to download the full image). In most cases, the binary delta will be much smaller.
 
 Once the delta (difference between the old and new image) is calculated, the device downloads and applies this delta onto the old application image, in place. When the process is finished, the new container image (new version of the application) is started.
 
 These binary deltas save on the amount of data needed to be downloaded, reduce the storage space requirements on the device to perform an application update, and shorten the time when Docker is updating.
 
 ## Enabling delta updates
 
-Delta update behaviour is enabled with the `RESIN_SUPERVISOR_DELTA` configuration variable:
+For newly created applications, the delta update behavior is enabled by default. For existing applications, the delta update behavior is enabled with the `RESIN_SUPERVISOR_DELTA` configuration variable:
```

Comment:
They might need to also add `RESIN_SUPERVISOR_DELTA_VERSION` == 3, to enable the V3 version of the deltas. @dfunckt can confirm

Reply:
Users can't choose delta versions anymore @shaunmulligan -- that config var is now inaccessible by users.